docker/runtime/nodejs-6.10/http-trigger/Dockerfile

```diff
@@ -1,11 +0,0 @@
-FROM node:6.10-alpine
```

Comment:
It has been moved to the parent folder (to avoid having a duplicated Dockerfile)